build_scripts/windows/scripts/build.cmd

```diff
@@ -134,6 +134,7 @@ for %%a in (%CLI_SRC%\azure-cli %CLI_SRC%\azure-cli-core %CLI_SRC%\azure-cli-tel
    %BUILDING_DIR%\python.exe -m pip install --no-warn-script-location --no-cache-dir --no-deps .
    popd
 )
+%BUILDING_DIR%\python.exe -m pip install setuptools
```

Comment:
Although `setuptools` is added to `azure-cli `dependency list in https://github.com/Azure/azure-cli/pull/27196, `azure-cli` is installed with `--no-deps`. Need to install it manually.